src/common/components/exchange/TxSummaryFixed.vue

```diff
@@ -603,10 +607,27 @@ export default defineComponent({
     const amountUSD = computed((): string => {
       const btcAmount = new SatoshiBig(props.summary?.amountFromString || 0, 'btc');
       if (!btcAmount || !bitcoinPrice) return VALUE_INCOMPLETE_MESSAGE;
-      // TODO: check casting accuracy
       return btcAmount.toUSDFromBTCString(bitcoinPrice.value, fixedUSDDecimals);
     });
 
+    const safeFeeUSD = computed((): string => {
+      const feeAmount = new SatoshiBig(safeFee.value.toString() || 0, 'btc');
+      if (!feeAmount || !bitcoinPrice) return VALUE_INCOMPLETE_MESSAGE;
```

Comment:
We have to access to the ref's value
```suggestion
      if (!feeAmount || !bitcoinPrice.value) return VALUE_INCOMPLETE_MESSAGE;
```

Reply:
changed